lib/android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -8,6 +8,7 @@
 import android.graphics.Point;
 import android.graphics.PorterDuff;
 import android.os.Build;
+import android.os.Handler;
```

Comment:
@rborn It looks like this file has nothing to do with the feature. Would it be correct just to revert this?